dns/dyndns/src/etc/inc/plugins.inc.d/dyndns/phpDynDNS.inc

```diff
@@ -42,6 +42,7 @@
  *    - regfish IPv6 (regfish.de)
  *    - dynv6 IPv6 (dynv6.com)
  *    - DigitalOcean (digitalocean.com)
+ *    - Gandi LiveDNS (gandi.net)
```

Comment:
There is also a "last tested" entry below. Can you fill in the date? :)

Reply:
Sure. Inserted it.